frontend/src/views/LoginPageView.tsx

```diff
@@ -12,44 +14,82 @@ import {
 } from "@/components/ui/card";
 import { Input } from "@/components/ui/input";
 import { Label } from "@radix-ui/react-label";
+import profileIcon from "@/assets/profile.png";
```

Comment:
could consider importing [lucide-react icons](https://lucide.dev/icons/) for efficiency instead of downloading asset pngs

```
import { Check } from "lucide-react"; // import check/tick icon

<Alert>
  <Check className="h-4 w-4" /> // use in frontend
  <AlertTitle>my alert title</AlertTitle>
  <AlertDescription>my alert description</AlertDescription>
</Alert>
```
![image](https://github.com/user-attachments/assets/c0e3963e-aca4-49f7-a8fd-455ce68356cd)